backend/docs/SETUP_COMPOSE.md

```diff
@@ -59,6 +59,13 @@ docker-compose exec web python manage.py createsuperuser
 docker-compose -f docker-compose.dev.yml down
 ```
 
+If you want to also purge your testing database, stop the containers and remove the corresponding volumes in the host
+
+```bash
+docker-compose -f docker-compose.dev.yml down --rmi all --volumes
```

Comment:
Wondering why we need `--rmi all` here to purge the database?

Reply:
Hmm these two commands haven't purged the ohshown event I added manually. Still can see it after ran `docker-compose -f docker-compose.dev.yml up -d` again.